src/docfx.website.themes/common/common.js

```diff
@@ -118,3 +119,30 @@ function getOverrideTemplate(uid) {
     content += "\n";
     return content;
 }
+
+function mergeSeeAlso(model) {
+    var seealso = [];
```

Comment:
Got your idea... should be a new list.

---
In reply to: [69418318](https://github.com/dotnet/docfx/pull/456#discussion_r69418318) [](ancestors = 69418318)